thunderbird-development/building-thunderbird/linux-build-prerequisites.md

```diff
@@ -4,41 +4,97 @@ description: >-
   environment set up and ready to hack on Thunderbird.
 ---
 
-# Linux Build Prerequisites
-
-This guide assumes you already followed the [Getting Started](../getting-started.md) documentation and you already downloaded `mozilla-central` and `comm-central` source code.
+# Hardware Requirements
 
 ## 64-bit version
 
-You need a 64 bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
+You will need to be running a 64-bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
 
 ```
 uname -m
 ```
 
 if this command returns `x86_64` you can proceed.
 
+## 30 GB of free space
+
+The Thunderbird build can use 30-40GB of disk space to complete depending on your operating system.
+
+{% hint style="warning" %}
+Note that while it's not technically required to have an internet connection to build, the default when building from `mozilla/comm-central` is that `--enable-bootstrap` is set so that the toolchains download automatically. If you do not have an active internet connection then 
```

Comment:
This sentence doesn't read very well, and it also looks incomplete.

Reply:
Fixed